includes/apple-exporter/class-workspace.php

```diff
@@ -92,7 +92,7 @@ public function write_json( $content ) {
 		// JSON should be decoded before being stored.
 		// Otherwise, stripslashes_deep could potentially remove valid characters
 		// such as newlines (\n).s
-		$decoded_json = json_decode( sanitize_text_field( $json ) );
+		$decoded_json = json_decode( $json );
```

Comment:
It's changing because we're adding support for HTML, so if you run `sanitize_text_field` on JSON that contains HTML it hoses it. The HTML output is being run through `wp_kses` with a strict set of tag and attribute restrictions.